src/runtime/virtcontainers/clh.go

```diff
@@ -747,15 +747,15 @@ func (clh *cloudHypervisor) toGrpc(ctx context.Context) ([]byte, error) {
 	return nil, errors.New("cloudHypervisor is not supported by VM cache")
 }
 
-func (clh *cloudHypervisor) Save() (s persistapi.HypervisorState) {
+func (clh *cloudHypervisor) Save() (s hv.HypervisorState) {
```

Comment:
Did HypervisorState already exist and persistapi have a copy. I did not see any code motion moving the type around?

Reply:
That's right -- I didn't remove it from persist API yet, that's why you only saw the addition. Let me go ahead and remove in this commit, too.